recipes/mayavi/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "4.5.0" %}
+
+package:
+  name: mayavi
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/enthought/mayavi.git
+  git_rev: {{ version }}
+
+build:
+  entry_points:
+    - mayavi2 = mayavi.scripts.mayavi2:main
+    - tvtk_doc = tvtk.tools.tvtk_doc:main
+  skip: True  # [osx or (linux and py34) or win]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - numpy
```

Comment:
Yeah, You do need to add `numpy x.x` then.

> By the way, seems that mayavi and tvtk could be in two differents packages.

Maybe you could test the new `conda-build` feature that creates multiple packages from a single recipe. (But I would not try that now. I would get this merged first ;-)